pkg/cmd/ssh_gcp.go

```diff
@@ -71,9 +77,13 @@ func sshToGCPNode(nodeName, path, user, pathSSKeypair string, sshPublicKey []byt
 	time.Sleep(45 * time.Second)
 
 	key := filepath.Join(pathSSKeypair, "key")
-	sshCmd := fmt.Sprintf("ssh -i " + key + " -o \"ProxyCommand ssh -W %%h:%%p -i " + key + " -o IdentitiesOnly=yes -o StrictHostKeyChecking=no " + bastionNode + "\" " + node + " -o IdentitiesOnly=yes -o StrictHostKeyChecking=no")
-	fmt.Println(sshCmd)
-	cmd := exec.Command("bash", "-c", sshCmd)
+	args := []string{"-i" + key, "-oProxyCommand=ssh -W%h:%p -i" + key + " -oIdentitiesOnly=yes -oStrictHostKeyChecking=no " + bastionNode, node, "-oIdentitiesOnly=yes", "-oStrictHostKeyChecking=no"}
+
+	if debugSwitch {
+		args = append([]string{"-vvv"}, args...)
+	}
+	fmt.Println(args)
```

Comment:
previously the whole ssh command was printed (including ssh in the beginning). Now you are printing just the args, which could be a bit out of context for the operator reading the gardenctl stdout and you are also printing it as array representation [arg1, arg2, ..] instead of args1 arg2 ...

That's why my suggestion was which should nicely print out the command
```go
fmt.Println("ssh " + strings.Join(args[:], " "))
```

Reply:
fixed, forgot to change it back as it uses for debug.